Annotato/Annotato/Persistence/PersistenceService/OnlinePersistenceService/OnlinePersistenceService+DocumentsPersistence.swift

```diff
@@ -52,6 +52,10 @@ extension OnlinePersistenceService: DocumentsPersistence {
         return deletedDocumentLocal
     }
 
+    func createOrUpdateDocument(document: Document) -> Document? {
+        localPersistence.documents.createOrUpdateDocument(document: document)
```

Comment:
Same thing as previous comment for lines 55 and 59.

```suggestion
        remotePersistence.documents.createOrUpdateDocument(document: document)
```